fix: minor updates

meruyert93 opened the pull request:
Hi Jonathan,
very nice solution 👍 
Here are just minor updates from my side:
1) code formatting for more readability 
2) updated `var` with `let` and `const`

meruyert93 committed:
fix: minor updates

jonTrent commented:
Yeah, good idea to leverage ES6 variable definitions rather than the antiquated loosely scoped "var".  Will review, tweak, test, and merge within the week...

jonTrent committed:
Update PatienceDiff.js

Tweaked proposed edits in function patienceDiffPlus to properly align declaration and use of variable "lastLineCountMoved".

Also, used https://zz85.github.io/mrdoobapproves/ to make code more readable.

jonTrent committed:
Update PatienceDiff.js

jonTrent reviewed (approved):
Merging in ECMA6 tweaks (ie, making use of "const" and "let" rather than "var").

There are no changes to the function logic whatsoever.
